src/main/java/seedu/address/model/DeckModelManager.java

```diff
@@ -18,28 +18,27 @@
 /**
  * Represents the in-memory model of the deck data.
  */
-public class CardModelManager implements CardModel {
-    private static final Logger logger = LogsCenter.getLogger(CardModelManager.class);
-
-    private final PowerDeck deck;
+public class DeckModelManager implements DeckModel {
+    private static final Logger logger = LogsCenter.getLogger(DeckModelManager.class);
+    private PowerDeck selectedDeck = null;
+    private final ArrayList<PowerDeck> powerDecks = new ArrayList<>();
```

Comment:
Should we make List of Decks into its own class? Like how the list of card is its own class that implements an Iterable

Reply:
Good point! I added it to a new issue #45 so we can close this PR first. For now, I just used ArrayList<PowerDeck>.